test/Error/aggregate_error.js

```diff
@@ -0,0 +1,384 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+   {
+       name: "proptotype and constructor",
+       body: function ()
+       {
+            assert.areEqual(AggregateError, AggregateError.prototype.constructor);
+            assert.isFalse(AggregateError.prototype.hasOwnProperty("errors"));
+            assert.areEqual("", AggregateError.prototype.message);
+            assert.areEqual("AggregateError", AggregateError.prototype.name);
+            assert.areEqual("object", typeof AggregateError.prototype);
+            assert.areEqual(Error.prototype, Object.getPrototypeOf(AggregateError.prototype));
+            assert.areEqual("AggregateError", AggregateError.name);
+            assert.areEqual(2, AggregateError.length);
+        }
+    },
+    {
+        name: "errors iterable to list failures",
+        body: function ()
+        {
+            class TestError extends Error {}
+
+            var case1 = {
+                get [Symbol.iterator]() {
+                    throw new TestError();
+                }
+            };
+            assert.throws(() => {
+                var obj = new AggregateError(case1);
+            }, TestError);
+
+            var case2 = {
+                get [Symbol.iterator]() {
+                    return {};
+                }
+            };
+            assert.throws(() => {
+                var obj = new AggregateError(case2);
+            }, TypeError);
+
+            var case3 = {
+                [Symbol.iterator]() {
+                    throw new TestError();
+                }
+            };
+            assert.throws(() => {
+                var obj = new AggregateError(case3);
+            }, TestError);

```

Comment:
Is this a repeat of case1?

Reply:
Nope. case1 is getter and this one is property.